basil/HL/mercury.py

```diff
@@ -16,7 +16,7 @@ class Mercury(HardwareLayer):
     '''Driver for the Physiks Instruments Mercury Controller.
     A protocoll via RS 232 serial port is used with 9600/19200/38400/115200 baud rate. The baud rate
     can be set with dip switches, as well as a hardware address to distinguish several
-    devices on one line. The write termination is CR and the read ends with '\r', '\n', '\x03'.
+    devices on one line. Keep in mind that the address that is set via the DIP switches corresponds to (board number + 1). Check p.45 of the overall manual below on how to properly set the address of each controller and what the corresponding board number is. The write termination is CR and the read ends with '\r', '\n', '\x03'.
```

Comment:
We do not enforce line break rules, but can you change this to reflect the line length of the others, please?